### PR TITLE
Shotgun pooling bugfix

### PR DESCRIPTION
**Pull request by tanaes**
I actually exercised this code in the real world last night. It worked great! But I found some things to change. 

1. quite to my surprise the qPCR machine output some nans for three of the blanks, which downstream functions weren't tolerating. 

2. It turns out our actual default qPCR dilution is 1:10,000 and not 1:25,000. Updated accordingly.

### CODE REVIEW

File: knimin/lib/tests/test_shotgun.py [hunk 1]
```diff
@@ -24,7 +24,21 @@ def setUp(self):
...
+            np.array([[39.25850585, 195.12485652, 193.73923464, 0.87336277],
+                      [199.34146596, 171.63359148, 160.97081284, 56.0640694],
+                      [8.48213356, 233.17824124, 292.90620164, 3.0180584]])
+
+        self.qpcr_conc_nan = \
+            np.array([[39.25850585, 195.12485652, 193.73923464, 0.87336277],
+                      [199.34146596, 171.63359148, 0, 56.0640694],
+                      [8.48213356, 233.17824124, 0, 3.0180584]])
+
+        self.qpcr_conc_25000 = \
```
Comment (wasade) [inline knimin/lib/tests/test_shotgun.py]: given the default change with `compute_qpcr_concentration`, should this variable name be 10000?

File: knimin/lib/tests/test_shotgun.py [hunk 2]
```diff
@@ -68,6 +82,18 @@ def test_compute_qpcr_concentration(self):
 
         npt.assert_allclose(obs, exp)
 
+    def test_compute_qpcr_concentration_nan(self):
+        obs = compute_qpcr_concentration(self.cp_vals_nan)
+        exp = self.qpcr_conc_nan
+
+        npt.assert_allclose(obs, exp)
+
+    def test_compute_qpcr_concentration_dil_25000(self):
```
Comment (wasade) [inline knimin/lib/tests/test_shotgun.py]: similar comment

Comment (wasade) [inline knimin/lib/tests/test_shotgun.py]: oh, i see, the 25000 makes sense here

**Comment by wasade** [comment]
Actual failures

https://travis-ci.org/biocore/labadmin/builds/224458051#L3341

**Comment by tanaes** [comment]
This should fix the failures. The DB function that was testing the addition of qPCR data `test_post` in `test_pm_shotgun_pool_handlers.py`) was actually relying on nans breaking the calculation output. 😝 I have updated that test accordingly, hopefully this will pass!

**Comment by tanaes** [comment]
Actually passing, hooray!

**Comment by wasade** [comment]
I think this is all superseded by labman.

**Pull request closed without merging**